python/cudf/cudf/tests/test_s3.py

```diff
@@ -269,6 +269,28 @@ def test_read_parquet_ext(
     assert_eq(expect, got1)
 
 
+def test_read_parquet_filesystem(s3_base, s3so, pdf):
+    fname = "data.0.parquet"
+    # NOTE: Need a unique bucket name when a glob pattern
+    # is used, otherwise fsspec seems to cache the bucket
+    # contents, and later tests using the same bucket name
+    # will fail.
+    bucket = "test_read_parquet_filesystem"
```

Comment:
*suggestion*: Shall we use a `uuid` here? Perhaps the s3_context could generate a unique bucket name and return it, so we could write:

```
with s3_context(base=..., files=...) as bucket:
   ...
```
WDYT?

Reply:
I think this is a good idea. Maybe I can do this in a follow-up PR since I'll need to touch most of the tests in this file?

Reply:
Sure.